applications/nrf_desktop/doc/smp_custom.rst

```diff
@@ -0,0 +1,81 @@
+.. _nrf_desktop_smp_custom:
+
+Custom Simple Management Protocol module
+########################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The custom |smp| is responsible for performing a Device Firmware Upgrade (DFU) over Bluetooth® LE.
+
+You can perform the DFU using for example the `nRF Connect for Mobile`_ application.
+The :guilabel:`DFU` button appears in the tab with the connected Bluetooth® devices.
+After pressing the button, you can select the :file:`*.bin` file that is to be used for the firmware update.
```

Comment:
Pleae copy information about the used bin file from our main doc page or link there (users may not know which bin file to actually copy here)